### PR TITLE
Javascript/javascript3/week1

### PR DESCRIPTION
**Pull request by valmdz**

### CODE REVIEW

File: javascript/javascript3/week1/script.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  )}&units=metric&appid=b8e2bab689f2c4f76b849b2533f789ff`;
+
+const button = document.querySelector("button");
+const input = document.querySelector("input");
+const weatherInfo = document.querySelector(".weather-info");
+
+const displayWeather = (evt) => {
+  evt.preventDefault();
+  if (input.value.trim() === "") {
+    weatherInfo.hidden = true;
```
Comment (priyankapandey13) [inline javascript/javascript3/week1/script.js:13]: If I don't write anything in the input box I would like to see a message which tells me that I need to input the city name. Right now it does nothing to let the user know that it needs value.

File: javascript/javascript3/week1/script.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return;
+  }
+  fetch(getUrl(input.value))
+    .then((response) => response.json())
+    .then((data) => {
+      weatherInfo.hidden = false;
+      const $icons = document.querySelector(".icons");
+      $icons.innerHTML = "";
+      for (const weather of data.weather) {
+        const $icon = document.createElement("img");
```
Comment (priyankapandey13) [inline javascript/javascript3/week1/script.js:23]: I would like to suggest some error handling here. 
If i test the code with dummy values except telling me to input some actual city name it breaks and show blank space.
feedback

In this second screenshot i searched for copenhagen and then put some fake value. I would like to see some message maybe asking for actual city and refreshing the old output.
feedback2

File: javascript/javascript3/week1/weather.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ <title>Document</title>
+</head>
+
+<body>
+ <h1>Weather <br> now</h1>
+ <form class='search'>
+  <input placeholder='Enter a city'></input>
+  <button type="submit">Get weather</button>
+ </form>
+ <div hidden class='weather-info'>
```
Comment (priyankapandey13) [inline javascript/javascript3/week1/weather.html:20]: Suggestion : You could have created this part in js file that way you will have more control over data to modify.

**Review by priyankapandey13 — CHANGES_REQUESTED** [review]
Hey Valentina, good work, I have few seggestion that you can work on to make your code better. And hope you are woring on the Giphy api because that was not mendatory.

File: javascript/javascript3/week1/script.js (hunk 1 above)
Comment (valmdz) [reply javascript/javascript3/week1/script.js:17]: Ok, thank you!

File: javascript/javascript3/week1/script.js (hunk 2 above)
Comment (valmdz) [reply javascript/javascript3/week1/script.js:37]: True, thank you for pointing that out

**Pull request merged**